# origin: Debian, #266594
# cset: n/a (imported into parisc-linux CVS)
# inclusion: already imported into parisc-linux CVS
# description: Fix unterminated comment
# revision date: 2004-09-03

--- a/include/asm-parisc/som.h	2004-08-19 13:54:14.000000000 -0700
+++ b/include/asm-parisc/som.h	2004-08-19 13:54:46.000000000 -0700
@@ -5,4 +5,4 @@
 #include <linux/som.h>
 
 
-#endif /* _ASM_PARISC_SOM_H
+#endif /* _ASM_PARISC_SOM_H */
